.vscodeignore

```diff
@@ -12,6 +12,4 @@
 !images/**/*
 !out/**/*.js
 !out/**/*.css
-!snippets.code-snippets
-
-!node_modules
```

Comment:
Was this intentional to remove `node_modules` from the `.vscodeignore`? Any reason to do it as part of this change?